lint/src/main/kotlin/com/google/samples/apps/nowinandroid/lint/designsystem/DesignSystemDetector.kt

```diff
@@ -50,7 +51,7 @@ class DesignSystemDetector : Detector(), Detector.UastScanner {
             }
 
             override fun visitQualifiedReferenceExpression(node: UQualifiedReferenceExpression) {
-                val name = node.receiver.asRenderString()
+                val name = node.receiver.tryResolveNamed()?.name ?: return
```

Comment:
This change is necessary for the `TestMode.FULLY_QUALIFIED` and `TestMode.IMPORT_ALIAS` to succeed